crm_salesperson_planner_sales/models/crm_salesperson_planner_visit.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 Sygel - Valentin Vinagre
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class CrmSalespersonPlannerVisit(models.Model):
+    _inherit = "crm.salesperson.planner.visit"
+
+    order_ids = fields.One2many("sale.order", "visit_id", string="Orders")
+    sale_order_count = fields.Integer(
+        compute="_compute_sale_data", string="Number of Sale Orders", store=True
+    )
+    quotation_count = fields.Integer(
+        compute="_compute_sale_data", string="Number of Quotations", store=True
+    )
+
+    @api.depends("order_ids.state")
+    def _compute_sale_data(self):
+        for sel in self:
```

Comment:
Use `read_group` for performance.

Reply:
I would say that's it, if not ... I could do it with a filtered using the first way.